lib/web/fetch/index.js

```diff
@@ -1449,14 +1449,21 @@ async function httpNetworkOrCacheFetch (
   //    13. Append the Fetch metadata headers for httpRequest. [FETCH-METADATA]
   appendFetchMetadata(httpRequest)
 
-  //    14. If httpRequest’s header list does not contain `User-Agent`, then
+  //    14. If httpRequest’s initiator is "prefetch", then set a structured
+  //    field value given (`Sec-Purpose`, the token 'prefetch') in httpRequest’s
+  //    header list.
```

Comment:
The comments were modified and do not match the spec anymore. following steps are incremented and do not match the spec.

![image](https://github.com/user-attachments/assets/b528f282-43c1-4edd-93d6-d0ef7310011b)

Reply:
What's the change suggested in this comment?
I've posted step 14 in while implementing this PR and incremented numbers in subsequent comments

Reply:
revert the comment changes.

Reply:
But the doc comment changes match the latest version of citations from the spec 🤔 

i.e. the changes in the PR match the current spec, while the existing source code has numbers from late 2022.

Reply:
let me have a second look ;)

Reply:
OH, my failure. Thank you for you patience.